utils/src/ooinstall/variants.py

```diff
@@ -11,6 +11,9 @@
 version.
 """
 
+import logging
+out = logging.getLogger('ooquickinstaller')
```

Comment:
You are correct. That is a typo. Note to self, fixit.

On Aug 22, 2016 1:05 PM, "Jason DeTiberus" <notifications@github.com> wrote:

> In utils/src/ooinstall/variants.py
> <https://github.com/openshift/openshift-ansible/pull/2341#discussion_r75748152>
> :
>
> > @@ -11,6 +11,9 @@
> >  version.
> >  """
> >
> > +import logging
> > +out = logging.getLogger('ooquickinstaller')
>
> Should this be ooquickinstaller when installer is used elsewhere?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/2341/files/e29035696f49533e1735cd176ea17842a42cb8e3#r75748152>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACBYQyX2GV1da1C8Cc4grEpEkHcljU5ks5qigEfgaJpZM4JqLOq>
> .
>